docs/current_docs/cookbook/cookbook.mdx

```diff
@@ -42,6 +41,13 @@ When working with private Git repositories, ensure that [SSH authentication is p
 ```php file=./snippets/copy-dir/php/src/MyModule.php
 ```
 
+</TabItem>
+
+<TabItem value="Java">
+
+```java file=./snippets/copy-dir/java/MyModule.java
```

Comment:
Not sure if this is an oversight but in other sections of the docs we have created a complete module and directory tree structure for each snippet. We should do the same here, as that will make it easier to implement automated testing in future.

Reply:
🤔 I think I saw that on some other docs section and as it was easier to do... But maybe I'm wrong.
I'll use complete module structure 👍